trinity/main.py

```diff
@@ -256,18 +252,6 @@ def trinity_boot(args: Namespace,
     networking_process.start()
     logger.info("Started networking process (pid=%d)", networking_process.pid)
 
-    main_endpoint.subscribe(
```

Comment:
@cburgdorf I've removed this as it seemed to be unnecessary

Reply:
We need this to enable plugins to shutdown trinity. This is currently used when running `trinity - - tx-pool - - light` for example.

Reply:
Bad formatting, mobile phone are terrible